src/sc_kpm/utils/creation_utils.py

```diff
@@ -8,8 +8,8 @@
 from sc_client.constants.sc_types import ScType
 from sc_client.models import ScAddr
 
-from sc_kpm import ScKeynodes
-from sc_kpm.common import CommonIdentifiers
+from sc_kpm.identifiers import CommonIdentifiers
+from sc_kpm.sc_keynodes import ScKeynodes
```

Comment:
Can we import ScKeynodes directly from sc_kpm?

Reply:
No, we can't because it brakes the order of imports. Also I think better to use full paths inside the lib